target/linux/ath79/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -16,7 +16,8 @@ ath79_setup_interfaces()
 			"0@eth1" "1:lan:4" "2:lan:1" "3:lan:2" "4:lan:3"
 		;;
 	mikrotik,routerboard-922uags-5hpacd|\
-	mikrotik,routerboard-wap-g-5hact2hnd)
+	mikrotik,routerboard-wap-g-5hact2hnd|\
+	mikrotik,routerboard-sxt-5nd-r2)
```

Comment:
if we are (were?) going to leverage the common platform, shouldn't this be `mikrotik,sxt5n`? i.e. the DTSI compatible, reflecting the fact that all devices share the same hardware? That was the whole point of the suggestion I was making. The DTSI compatible is otherwise completely useless.
This comment applies to all the other scripts.

Reply:
Maybe alternatively change the name of the dtsi to what is here?

Reply:
This should bear the DTS compatible name as done for all other devices, since introducing an inconsistency compared to all other DTS targets here would not be helpful.